frontend/src/pages/projects/screens/detail/storage/StorageTableRow.tsx

```diff
@@ -79,6 +107,47 @@ const StorageTableRow: React.FC<StorageTableRowProps> = ({
           </Flex>
           <Text>{getDescriptionFromK8sResource(obj)}</Text>
         </Td>
+
+        {isStorageClassesAvailable && (
+          <Td dataLabel="Storage class">
```

Comment:
the spacing is not the same as the other column

do what the name of the pvc does

```tsx
 <Flex
            spaceItems={{ default: 'spaceItemsSm' }}
            alignItems={{ default: 'alignItemsCenter' }}
          >
            <FlexItem>
              <TableRowTitleDescription title={getDisplayNameFromK8sResource(obj)} resource={obj} />
            </FlexItem>
            <FlexItem>
              <StorageWarningStatus obj={obj} onEditPVC={onEditPVC} onAddPVC={onAddPVC} />
            </FlexItem>
          </Flex>
```